src/governance/snapshot/snapshot.service.ts

```diff
@@ -71,15 +73,20 @@ export class SnapshotService {
       [vote.choice1]: vote.result1,
       [vote.choice2]: vote.result2,
       [vote.choice3]: vote.result3 ?? 0,
+      [vote.choice4]: vote.result4 ?? 0,
     };
     return Object.keys(choiceResults).reduce((a, b) =>
       choiceResults[a] > choiceResults[b] ? a : b,
     );
   }
 
   hasQuorum(vote: VoteEntity): boolean {
+    // if proposal is an approval proposal and has multiple choices (1a, 1b, 2a, 2b),
+    // count quorum percent using just the first choice (1a, 1b)
+    const additionalQuorumSubtrahend =
+      vote.proposalType === 'approval' ? 0 : (vote.result3 ?? 0) - (vote.result4 ?? 0);
     const percent =
-      Math.round(LDO_5_PERCENT_QUORUM - vote.result1 - vote.result2 - (vote.result3 ?? 0)) /
+      Math.round(LDO_5_PERCENT_QUORUM - vote.result1 - vote.result2 - additionalQuorumSubtrahend) /
```

Comment:
```suggestion
      Math.round(LDO_5_PERCENT_QUORUM - vote.result1 - vote.result2 - additionalQuorumSubtrahend) /
```
Different logic for calculation